.github/workflows/build-mobile.yml

```diff
@@ -292,3 +168,106 @@ jobs:
           key: ${{ runner.os }}-pods-${{ hashFiles('apps/ledger-live-mobile/ios/Podfile.lock') }}
       - name: install dependencies
         run: pnpm i --filter="live-mobile..." --filter="ledger-live" --no-frozen-lockfile --unsafe-perm
+
+  report:
+    runs-on: ubuntu-latest
+    needs: [build-mobile-app-android, build-mobile-app-ios]
+    if: always() && !cancelled() && github.event_name == 'workflow_dispatch' && github.event.inputs.sha != ''
+    steps:
+      - uses: actions/github-script@v6
+        name: get statuses of runs
+        id: status
+        with:
+          script: |
+            const fs = require("fs");
```

Comment:
same here also if some code is duplicated it might be a good idea to factorise it, however it depends.